lib/miq_expression/field.rb

```diff
@@ -25,8 +25,11 @@ def date?
     column_type == :date
   end
 
-  def self.valid_field?(field)
-    FIELD_REGEX.match(field).present? if field.kind_of?(String)
+  def self.is_field?(field)
```

Comment:
I think I prefer `.field?`, which will also keep rubocop from complaining. Unless you feel strongly!